column2Vec/reaserch/generate_report.py

```diff
@@ -0,0 +1,358 @@
+            "condition5": {"relationship": "YES", "value": 0.6},
+            "reg_city1": {"relationship": "NO", "value": 0.35},
+            "reg_state1": {"relationship": "NO", "value": 0.35},
+            "country5": {"relationship": "NO", "value": 0.35},
+            "country7": {"relationship": "NO", "value": 0.35},
+            "Star46": {"relationship": "NO", "value": 0.35},
+            "Star16": {"relationship": "NO", "value": 0.35},
+            "make3": {"relationship": "NO", "value": 0.4},
+            "car_name4": {"relationship": "NO", "value": 0.35},
+            "fuel_type4": {"relationship": "NO", "value": 0.35},
+            "fuel3": {"relationship": "NO", "value": 0.35},
+            "flight1": {"relationship": "NO", "value": 0.4},
+            "tail_number1": {"relationship": "NO", "value": 0.4}
+        },
+        "fuel_type4": {
+            "fuel3": {"relationship": "YES", "value": 0.7},
+            "flight1": {"relationship": "NO", "value": 0.35},
+            "condition5": {"relationship": "NO", "value": 0.35},
+            "tail_number1": {"relationship": "NO", "value": 0.35},
+            "date_added7": {"relationship": "NO", "value": 0.35}
+        },
+        "fuel3": {
+            "fuel_type4": {"relationship": "YES", "value": 0.7},
+            "flight1": {"relationship": "NO", "value": 0.35},
+            "condition5": {"relationship": "NO", "value": 0.35},
+            "tail_number1": {"relationship": "NO", "value": 0.35},
+            "date_added7": {"relationship": "NO", "value": 0.35}
+        },
+        "flight1": {
+            "tail_number1": {"relationship": "YES", "value": 0.7}
+        },
+        "tail_number1": {
+            "flight1": {"relationship": "YES", "value": 0.7}
+        }
+    }
+
+    # with open(f"files/{file_name}similar.md", "w") as f_sim:
+    #     with open(f"files/{file_name}not_similar.md", "w") as f_not:
+    #         for function in test_results.keys():
+    #             f_sim.write(f"## Function: {function} \n")
+    #             f_not.write(f"## Function: {function} \n")
+    #             written_column = []
+    #             for column in test_results[function].keys():
+    #                 if column[0] not in columns_to_test or column[1] not in columns_to_test:
+    #                     continue
+    #                 if column[0] not in written_column:
+    #                     f_sim.write(f"\n### {column[0]}: \n")
+    #                     f_not.write(f"\n### {column[0]}: \n")
+    #                     written_column.append(column[0])
+    #                 if test_results[function][column] > 0.5:
+    #                     f_sim.write(f"{column[1]} ({test_results[function][column]}),  ")
+    #                 else:
+    #
+    #                     f_not.write(f"{column[1]} ({test_results[function][column]}),  ")
+    #             f_sim.write("\n")
+    #             f_not.write("\n")
+    #         f_sim.write("\n")
+    #         f_not.write("\n")
```

Comment:
Commented out code either has to have a good explanation next to it or should not be there.

Reply:
Fixed